extensions/amp-story/amp-story.md

```diff
@@ -331,6 +335,26 @@ A "poster" is an image that displays in the UI until your story is loaded. The p
 
 The `<amp-story>` component contains one or more [`<amp-story-page>`](#pages:-amp-story-page) components, containing each of the individual screens of the story.  The first page specified in the document order is the first page shown in the story.
 
+### Landscape orientation and full bleed desktop experience opt in
+
+If the `supports-landscape` attribute is specified on the `<amp-story>` element, it will:
```

Comment:
Is there a way to specify different assets for landscape vs portrait view? What is the behavior if a portrait image is included and then the landscape/full beed is enabled? Will it distort it?

Reply:
Good point!
Publishers are already specifying assets that work for different sizes without being distorted: phones and tablets have all kind of screen sizes/ratio.
Right now we expect publishers to either provide assets that work on both landscape and portrait (like some have been already doing), or write some CSS using media queries to figure out what to show/hide themselves.
We're planning on introducing more templating options later, exactly like you're suggesting.

Do you think we should mention that, or only add it to the documentation once these new features are ready?

Reply:
I feel like we should *not* recommend the media queries, as it's actually rather difficult to get them to work properly and I'm not sure we want to consider it an officially-supported thing to do.  In my opinion, we now officially support center-crop only, and we'll support specifying different media for different orientations at a later date by having an `<amp-story>` primitive of some sort to do so.

Reply:
Sounds good, thanks. So we don't add anything to the documentation for now, and document our solution once we implement one?

Reply:
Sounds good to me

Reply:
sounds good, thanks for clarifying!